ansible/roles/ocp-workload-cnad-app-modernization/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+ocp_username: snandaku-redhat.com
+ocp_user_needs_quota: True
+
+ocp_user_groups:
+  - OPENTLC-PROJECT-PROVISIONERS
+
+quota_requests_cpu: 8
```

Comment:
I assume this workload is for OCP 4. If so the role should be called `ocp_workload_cnad_app_modernization`.

Also all variables (except ocp_username) should start with the name of the workload to make it obvious that they are for this role.

So for example `ocp_workload_cnad_app_modernization_quota_requests_cpu`.

Reply:
ocp4-workload-cnad-app-modernization ?